Example/Tests/MedStickerEncryptionTests.swift

```diff
@@ -174,4 +174,40 @@ final class MedStickerEncryptionTests: XCTestCase {
     // then
     XCTAssertTrue(signature.hasPrefix("adam-sha256"))
   }
+
+  func testGenerateFingerprintSecret_charlie__shouldGenerate132StringsAndContainVersionCharlie() {
+    let expectedLength = 64 * 2 + 8 // 64 bytes * 2(as hex string) + "charlie:" (8)
+
+    let fakePin = "fakePin"
```

Comment:
```suggestion
    let fakePin = UUID().uuidString
```
better to have random to reduce the change of false positivity